LoopKit.xcodeproj/project.pbxproj

```diff
@@ -2374,10 +2378,23 @@
 			children = (
 				E9077D2624ACD59F0066A88D /* InformationView.swift */,
 				E9077D2924ACDE2C0066A88D /* CorrectionRangeInformationView.swift */,
+				E916F56E24AE2FFE00BE3547 /* CorrectionRangeOverrideInformationView.swift */,
 			);
 			path = "Information Screens";
 			sourceTree = "<group>";
 		};
+		E916F56724AD31F900BE3547 /* Settings Editors */ = {
```

Comment:
I added a folder for the settings editors so they're easier to find within LoopKitUI

Reply:
Thanks!